peachjam/tasks.py

```diff
@@ -28,6 +28,29 @@ def update_document(ingestor_id, document_id):
     log.info("Ingestor is disabled, ignoring.")
 
 
+@background(queue="peachjam", schedule=(60 * 60 * 5), remove_existing_tasks=True)
```

Comment:
5 hours?? Maybe `60 * 5` is better, 5 mins?

Reply:
:laughing: